benchmark/benchmark_device_histogram.cpp

```diff
@@ -545,18 +553,24 @@ void add_even_benchmarks(std::vector<benchmark::internal::Benchmark*>& benchmark
     };
 }
 
-#define CREATE_MULTI_EVEN_BENCHMARK(CHANNELS, ACTIVE_CHANNELS, T, BINS, SCALE) \
-benchmark::RegisterBenchmark( \
-    (std::string("multi_histogram_even") + "<Channels:" #CHANNELS ",Active Channels:" #ACTIVE_CHANNELS ",Datatype:" #T ">" + \
-        "(Entropy Percent:" + std::to_string(get_entropy_percents(entropy_reduction)) + "%,Bin Count:" + \
-        std::to_string(BINS) + " bins)" \
-    ).c_str(), \
-    [=](benchmark::State& state) { \
-        run_multi_even_benchmark<T, CHANNELS, ACTIVE_CHANNELS>( \
-            state, BINS, SCALE, entropy_reduction, stream, size \
-        ); \
-    } \
-)
+#define CREATE_MULTI_EVEN_BENCHMARK(CHANNELS, ACTIVE_CHANNELS, T, BINS, SCALE)  \
+    benchmark::RegisterBenchmark(                                               \
+        std::string("device_multi_histogram_even"                               \
+            "<channels:" #CHANNELS                                              \
+            ",active_channels:" #ACTIVE_CHANNELS                                \
+            ",data_type:" #T ">."                                                \
+            "(entropy_percent:"                                                 \
```

Comment:
Might need to re-run clang-format to re-align the `\`

Reply:
Ran with .clang-format file provided in project directory in latest commit